client/asset/btc/walletclient.go

```diff
@@ -154,31 +153,6 @@ func (wc *walletClient) PrivKeyForAddress(addr string) (*btcec.PrivateKey, error
 	return wif.PrivKey, nil
 }
 
-// SignMessage attempts to have the wallet sign the message with the private key
-// associated with the specified address.
-func (wc *walletClient) SignMessage(addr string, msg dex.Bytes) (pubkey, sig dex.Bytes, err error) {
-	var keyHex string
-	err = wc.call(methodPrivKeyForAddress, anylist{addr}, &keyHex)
-	if err != nil {
-		return nil, nil, err
-	}
-	wif, err := btcutil.DecodeWIF(keyHex)
-	if err != nil {
-		return nil, nil, err
-	}
-	pubkey = wif.PrivKey.PubKey().SerializeCompressed()
-	var b64Sig string
-	err = wc.call(methodSignMessage, anylist{keyHex, msg.String()}, &b64Sig)
-	if err != nil {
-		return nil, nil, err
-	}
-	sig, err = base64.StdEncoding.DecodeString(b64Sig)
```

Comment:
This was incorrect. The base-64 encoded signature uses a compact format that didn't work for our needs.